tests/test_calendar_model.py

```
@@ -0,0 +1,117 @@
+    Fields stored as empty strings must not produce empty properties.
+    Our ``field or None`` conversions guard against this.
+    """
+    cal = CalendarFactory(description="", color="", method="")
+    output = cal.to_ical()
+    parsed = Calendar.from_ical(output)
+    assert parsed.get("COLOR") is None
+    assert parsed.get("METHOD") is None
+
+
+@pytest.mark.django_db
+def test_to_ical_all_optional_fields_round_trip(db):
```

Ruff (0.15.2)

[warning] Unused function argument: `db`

(ARG001)